server/src/main/java/io/crate/planner/GCDanglingArtifactsPlan.java

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
Hm strange that git did not detect the rename correctly. Would be great to have this in the history, anyway, maybe re-check this next time (or try to re-apply the change but maybe not worth the effort).